apps/gradient_autoscheduler/Makefile

```diff
@@ -0,0 +1,53 @@
+include ../support/Makefile.inc
+
+WITH_PYTHON ?= 0
```

Comment:
Why does WITH_PYTHON default to zero?

Reply:
(at least add a comment explaining it)

Reply:
It defaults to 1 now. If we can't find pybind11 we prompt the user to set WITH_PYTHON to 0 if they wish to.